payment_ippay_ach/__manifest__.py

```diff
@@ -10,12 +10,13 @@
     "maintainer": "Open Source Integrators",
     "website": "http://www.opensourceintegrators.com",
     "category": "Accounting",
-    "depends": ["payment"],
+    "depends": ["payment", "website", "account_payment"],
```

Comment:
Can we remove the website dependency?

Reply:
yes, we do not need "payment", "website"